R-package/R/lgb.Predictor.R

```diff
@@ -142,6 +142,10 @@ Predictor <- R6::R6Class(
 
         # Check if data is a matrix
         if (is.matrix(data)) {
+        # Check whether matrix is the correct type first ("double")
+          if (storage.mode(data) != "double") {
```

Comment:
```suggestion
          # Check whether matrix is the correct type first ("double")
          if (storage.mode(data) != "double") {
```

align comment start with the code it describes